code/style.css

```diff
@@ -1,5 +1,173 @@
 /* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
 
+@import url("https://use.typekit.net/zkc3tby.css");
+
 body {
-  background-color: peachpuff;
+  background-color: #FFFF;
+  margin: 0;
+}
+
+header {
+  height: 50px;
+  background-color: #BCCECC;
+}
+
+header img {
+  width: 120px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+
+}
+
+h1 {
+  font-family: europa, sans-serif;
+  font-style: normal;
+  font-weight: 700; 
+  letter-spacing: 2px;
+  font-size: 25px;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+h2 {
+  font-family: europa, sans-serif;
+  font-style: normal;
+  font-weight: 500; 
+  letter-spacing: 1px;
+  font-size: 17px;
+  margin-top: -10px;
+
+}
+
+h3 {
+  font-family: europa, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: 1px;
+  font-size: 23px;
+  margin-top: -15px;
+  color: #BCCECC; 
+}
+
+p {
+  font-family: europa, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  letter-spacing: 1px;
+  font-size: 10px;
+  text-align: center;
+}
+
+.container {
+  width: 100%;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 25px;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 12px 0 0 30px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 3px;
+  background-color: #FFF;
+}
+
+nav {
+  display: none;
 }
+
+nav a {
+  color: #FFFF;
+  text-decoration: none;
+}
+
+.flex-container .container {
+  padding-top: 200px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+}
+
+.karten {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 20px 0 0 0;
+}
+
+.karten img {
+  width: 300px;
+}
+
+footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 75px;
+  margin: 100px 0 0 0;
+  background-color: #BCCECC;
+}
+
+.karten:hover  {
+  outline: #D9D9D9 solid 2px;
+}
+
+
+/*** Media Queries ***/
+
+@media (min-width: 880px) {
+
+  nav {
+    display: inline-flexbox;
+    height: 50px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+}
+
+@media (min-width: 880px) {
```

Comment:
You could wrap all of your code in one media queries since they have the same measurements!

Reply:
Also, this media query doesn't have a closing bracket in the end `}`